tools/udisksctl.c

```diff
@@ -943,8 +943,6 @@ read_passphrase (void)
   str = NULL;
 
  out:
-  if (str != NULL)
-    g_string_free (str, TRUE);
   return ret;
```

Comment:
I'd prefer the ``goto``s above to be replace by ``return``s with the ``out`` label removed instead. But YMMV.

Reply:
Fixed.